e4s_cl/cf/assets.py

```diff
@@ -1,8 +1,9 @@
 """
 This modules parses data in the asset dirs to list all assets available during execution
 """
-
+from pydoc import locate
 from e4s_cl import logger
+from e4s_cl.model.profile import attributes
```

Comment:
It would be cleaner to use `Profile.attributes` (a property), which amounts to the same but by importing the model instead of a random function used for the model